crates/router/src/lib.rs

```diff
@@ -20,10 +20,20 @@ use ya_sb_util::PrefixLookupBag;
 mod dispatcher;
 
 lazy_static! {
-    pub static ref GSB_PING_TIMEOUT: u64 = env::var("GSB_PING_TIMEOUT")
-        .unwrap_or("5".into())
+    static ref RAW_GSB_PING_TIMEOUT: u64 = env::var("GSB_PING_TIMEOUT")
+        .unwrap_or(20.to_string())
```

Comment:
This change should probably be discussed with @tworec

Reply:
currently the timeout is 10 ( due to the logic below checking X2 )

without reconnect the small linux machine dies within minutes.

@tworec what do you think of raising this timeout from 10 to 20 seconds?

Reply:
I agree
@prekucki suggested similar value as far as I remember our phone call